jdbc/src/test/java/org/jclouds/jdbc/BaseJdbcBlobStoreTest.java

```diff
@@ -0,0 +1,746 @@
+/*
```

Comment:
This test is the equivalent of FilesystemBlobStoreTest. Some of the test methods are different from the ones included in the integration tests and since I wanted to be sure everything was working I took these too. Should I remove these?